lib/network/modules/LayoutEngine.js

```diff
@@ -1743,11 +1743,13 @@ class LayoutEngine {
     if (nodeArray.length > 1) {
       if(this._isVertical()) {
         nodeArray.sort(function (a, b) {
+          if (a.x === undefined || b.x === undefined) return 0;
```

Comment:
How often do these sorting functions get used?

Might be worth extracting these sort functions out to top-level functions

Reply:
These will change in #3411

Reply:
K